app/controllers/users_controller.rb

```diff
@@ -65,4 +74,25 @@ def should_display_user?(user)
   def get_user
     @user = User.friendly.find(params[:id])
   end
+
+  def set_filter_params
+    filter_params = params.slice(:project_filter, :timezone_filter, :online)
+    set_timezone_degree_range(filter_params)
+    filter_params
+  end
```

Comment:
Also, it is not a good idea for a method called `set_xxx` to be redirecting anywhere :wink:, if the method is called `set_xxx`, it should do just that, set `xxx` to some new value.

On a different note, this method isn’t really just setting an instance variable (as one would expect from the method signature), it is really getting/computing the filter params. Perhaps it should just be called `filter_params`?